solutions/control flow/dice.js

```diff
@@ -0,0 +1,18 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+
+numberOnTheTopFace = parseInt(readLine());
+
+numberOnTheBottom = 7 - numberOnTheTopFace;
```

Comment:
@prateek-mehra 

1. Please use `let` to declare variables.
2. Please rename the folder to `control-flow`.